code/modules/mob/mob.dm

```diff
@@ -1203,7 +1203,10 @@
 /mob/proc/can_read(atom/viewed_atom, reading_check_flags = (READING_CHECK_LITERACY|READING_CHECK_LIGHT), silent = FALSE)
 	if((reading_check_flags & READING_CHECK_LITERACY) && !is_literate())
 		if(!silent)
-			to_chat(src, span_warning("You try to read [viewed_atom], but can't comprehend any of it."))
+			if(HAS_TRAIT_FROM(src, TRAIT_ILLITERATE, FARSIGHT_TRAIT)) //ORBSTATION: farsighted gives alternate text
```

Comment:
I guess you should leave it as it is in case of the edge case I mentioned above where you _are_ far sighted but are wearing glasses but have gained illiteracy from head trauma.